src/Files.App/Actions/Content/Install/InstallCertificateAction.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using Files.App.Commands;
+using Files.App.Contexts;
+using Files.Backend.Helpers;
+
+namespace Files.App.Actions;
+
+internal class InstallCertificateAction : ObservableObject, IAction
+{
+	private readonly IContentPageContext context = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+	public string Label => "Install".GetLocalizedResource();
+
+	public string Description => "InstallCertificateDescription".GetLocalizedResource();
+
+	public RichGlyph Glyph { get; } = new("\uEB95");
+	
+	public bool IsExecutable => context.SelectedItems.Count == 1 &&
```

Comment:
Should probably support multiple items as well

Reply:
I was afraid this could cause a crash, but it works great!